### PR TITLE
Clarifications and style edits for the tutorial

### LINKED ISSUE #4554: Run through the Getting started guide and create a friction log
There have been a few issues coming up about the getting started guide. 

- [x] follow the guide and create a friction log
- [x] make fixes or follow up issues where necessary

### LINKED ISSUE #4736: Wrong output on traffic split example tab
**Describe the change you'd like to see**
https://knative.dev/docs/getting-started/first-traffic-split/#splitting-traffic
The spected output when you click the kubectl tab is wrong, it keeps the same output as the one on the kn CLI tab, i.e. it shows the TRAFFIC column that is specific to kn CLI

### PR DESCRIPTION
**Pull request by snneji**
Fixes #4554
Fixes #4736 

- Style edits to all pages in the tutorial
- Added a step for running `minikube tunnel` to the quickstart install page
- Changed the name of some of the Eventing tutorial pages to make it clearer that the tutorial covers sources and sinks

### CODE REVIEW

File: docs/getting-started/first-traffic-split.md [hunk 1]
```diff
@@ -10,163 +10,179 @@ The last super power :rocket: of Knative Serving we'll go over in this tutorial
...
-    spec:
-      template:
+    1. Edit your existing `hello.yaml` file to contain the following:
+        ``` bash
+        ---
+        apiVersion: serving.knative.dev/v1
+        kind: Service
         metadata:
-          name: hello-knative
+          name: hello
```
Comment (csantanapr) [inline docs/getting-started/first-traffic-split.md:48]: Good fix

File: docs/getting-started/first-traffic-split.md [hunk 2]
```diff
@@ -10,163 +10,179 @@ The last super power :rocket: of Knative Serving we'll go over in this tutorial
...
-      name: hello
-    spec:
-      template:
+    1. Add the `traffic` section to the bottom of your existing `hello.yaml` file:
+        ``` bash
+        apiVersion: serving.knative.dev/v1
+        kind: Service
         metadata:
-          name: hello-knative
+          name: hello
```
Comment (csantanapr) [inline docs/getting-started/first-traffic-split.md:137]: Good catch

File: docs/snippets/quickstart-install.md [hunk 3]
```diff
@@ -118,3 +121,12 @@ To get a local deployment of Knative, run the `quickstart` plugin:
         ```bash
         minikube profile list
         ```
+    1. To finish setting up networking for minikube, start the `minikube tunnel` process in a separate terminal window:
```
Comment (csantanapr) [inline docs/snippets/quickstart-install.md:124]: Good addition

File: docs/getting-started/first-autoscale.md [hunk 4]
```diff
@@ -30,26 +34,27 @@ kubectl get pod -l serving.knative.dev/service=hello -w
...
 
-==**Expected output:**==
-```{ .bash .no-copy }
-NAME                                     READY   STATUS
-hello-world                              2/2     Running
-hello-world                              2/2     Terminating
-hello-world                              1/2     Terminating
-hello-world                              0/2     Terminating
-```
+!!! Success "Expected output"
```
Comment (csantanapr) [inline docs/getting-started/first-autoscale.md:37]: I like this style of green check

**Review by csantanapr — COMMENTED** [review]
thanks for all the improvements

File: docs/snippets/quickstart-install.md (hunk 3 above)
Comment (psschwei) [inline docs/snippets/quickstart-install.md:124]: @snneji  only Windows/Mac users need to manually do this step (discussion of what that's the case is in https://github.com/knative-sandbox/kn-plugin-quickstart/pull/139 )

Comment (snneji) [reply docs/snippets/quickstart-install.md:124]: @psschwei I'll add an issue to clarify this. 

If Linux users do this step, does it cause any issues?

**Pull request merged**